content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,168 @@
+# Data Management for Machine Learning
+
+<!--
+## Data Management for Machine Learning
+-->
+
+Where and how to store data and ML model files is one of the first decisions
+your team will face. But traditional back-up strategies do not fit the data
+science lifecycle. Large files end up scattered throughout several buckets.
+Overlapping dataset versions coexist, causing leakage and inefficient use of
+space. The project evolution is harder to track. What was the name of the best
+model? Is it safe to delete `huge_split.zip`? Can others reproduce my results?
+
+![Direct access storage](/img/direct_access_storage.png) _The S3 bucket on the
+right is shared (and bloated) by several people and projects. You need to know
+the exact location of the correct files, and use cloud-specific tools (e.g. AWS
+CLI) to access them directly._
+
+To maintain control and visibility over all your data and models, DVC stores
+large files and directories for you in a structured way. It tracks them by
+logging their locations and unique descriptions in **code-like metafiles**.
+
+![DVC-cached storage](/img/dvc_managed_storage.png)
+
+![]() _DVC writes `.dvc` files next to large files. A data cache indexes them
+with `md5` checksums. Mass storage only holds unique files pushed with DVC for
+back up or sharing._
+
+Committing DVC metafiles to Git along your ML source files creates
+**reproducible project versions**: Its history becomes easy to review, rewind,
+and repeat going forward (by anyone). There's no need to come up with naming
+schemes for changed data and model files.
+
+![Versioning data with Git](/img/project_versioning.png) _You can use Git
+history to store different dataset and model versions without renaming any files
+in your workspace. The project cache grows as more relevant versions are
+tracked._
+
+## How it works
```

Comment:
It doesn't explain the remote storage, which should be also compatible with the new cloud versioning type of storage

Reply:
Remote storage is in a separate PR: https://github.com/iterative/dvc.org/pull/4058

Reply:
Added a tip admon about remote storage and cloud versioning. PTAL